src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2217,7 +2221,9 @@ because xbuild doesn't support framework reference assemblies.
 			RuntimeConfigBinFilePath="$(_BinaryRuntimeConfigPath)"
 			ZipFlushFilesLimit="$(_ZipFlushFilesLimit)"
 			ZipFlushSizeLimit="$(_ZipFlushSizeLimit)"
-			ExcludeFiles="@(AndroidPackagingOptionsExclude)">
+			ExcludeFiles="@(AndroidPackagingOptionsExclude)"
+      AndroidBinUtilsDirectory="$(AndroidBinUtilsDirectory)"
```

Comment:
jonp will mention the indentation here :P

Reply:
If only indentation in our .targets were consistent... :)

Reply:
Updated :)

Reply:
I'd be ok with doing a conversion of all spaces to tabs (or tabs to spaces). it would mess up the history a bit though.

Reply:
I've been in favor or this for quite a while now... git allows one to ignore whitespace changes in `git log` by passing the `-w` flag, so if it were just a single commit, it wouldn't have that bad of an effect.